src/main/java/reposense/commits/model/AuthorDailyContribution.java

```diff
@@ -8,16 +8,11 @@
  * Holds the commits made and line contribution count of an {@code Author} for a single day.
  */
 public class AuthorDailyContribution {
```

Comment:
Perhaps this class can be changed to `AuthorDailyCommits`?

Reply:
That would change too many entries and I feel it is not really necessary.

Actually if you want to change the name I feel a better idea is to keep `AuthorDailyContribution` and change `commitResults` to `commits`